bin/asaph_localize

```diff
@@ -328,7 +331,7 @@ def parseargs():
     
     subparsers = parser.add_subparsers(dest="mode")
 
-    plot_parser = subparsers.add_parser("manhattan-plot",
+     plot_parser = subparsers.add_parser("manhattan-plot",
```

Comment:
Similarly, does the addition of a space cause an indentation problem for Python?